sources/views/settings.js

```diff
@@ -0,0 +1,25 @@
+import { JetView } from "webix-jet";
+
+export default class Settings extends JetView {
+	config() {
+		const changeLanguageButtons = {
+			view: "segmented",
+			value: "EN",
+			options: [
+				{
+					localId: "en",

```

Comment:
localIds work with Webix views only. 
data item, control options - use regular ids, otherwide they will not work (generate auto ids in backgroud and we will not be able to work with this menu/control )

Reply:
Ready to review.